listenbrainz/webserver/utils.py

```diff
@@ -25,3 +31,22 @@ def reformat_date(value, fmt="%b %d, %Y"):
 
 def reformat_datetime(value, fmt="%b %d, %Y, %H:%M %Z"):
     return value.strftime(fmt)
+
+
+def inject_global_props():
+    current_user_data = {}
+    spotify_data = {}
+    if current_user.is_authenticated:
+        current_user_data = {
+            "id": current_user.id,
+            "name": current_user.musicbrainz_id,
+            "auth_token": current_user.auth_token,
+        }
+        spotify_data = spotify.get_user_dict(current_user.id)
+    props = {
+        "api_url": current_app.config["API_URL"],
```

Comment:
There's also the websockets url we can pass.
It's not used everywhere but seems like the sort of thing we would want in global props

Reply:
Makes sense, I'll add it to global props.

Reply:
I think there might be a useful side-effect with the naming convention difference, in that we can easily identify whether the value is coming directly from the parsed props text or if it has already been passed as a component property.